cg/server/api.py

```diff
@@ -364,21 +364,35 @@ def get_sequencing_metrics(flow_cell_name: str):
     """Return sample lane sequencing metrics for a flow cell."""
     if not flow_cell_name:
```

Comment:
change this to `flow_cell_id` to be clear

Reply:
Would change the signature, not sure how that impacts callers..